spectrocloud/resource_cluster_eks.go

```diff
@@ -162,10 +162,21 @@ func resourceClusterEks() *schema.Resource {
 							Default:      "public",
 						},
 						"public_access_cidrs": {
-							Type:     schema.TypeSet,
-							Optional: true,
-							ForceNew: true,
-							Set:      schema.HashString,
+							Type:        schema.TypeSet,
+							Optional:    true,
+							ForceNew:    true,
+							Set:         schema.HashString,
+							Description: "List of CIDR blocks that define the allowed public access to the resource. Requests originating from addresses within these CIDR blocks will be permitted to access the resource. All other addresses will be denied access.",
```

Comment:
what is the default value @SivaanandM ?

Reply:
we are not using this attribute for cluster it used in resource_workspace

Reply:
if we set default it always show as a change in day 2 operation

Reply:
we don’t have any default values for private_access_cidrs  and public_access_cidrs, its a list option